lib/matplotlib/tri/trirefine.py

```diff
@@ -51,8 +50,8 @@ class UniformTriRefiner(TriRefiner):
 
     Parameters
     ----------
-    triangulation : :class:`~matplotlib.tri.Triangulation`
-                     The encapsulated triangulation (to be refined)
+    triangulation : `.Triangulation`
```

Comment:
```suggestion
    triangulation : `~matplotlib.tri.Triangulation`
```
see doc policy, as above